.travis.yml

```diff
@@ -136,29 +105,23 @@ jobs:
 
     - name: "Run redeployment integration tests"
       stage: redeployment integration tests
-      if: commit_message !~ \[skip-integration-tests\] OR branch = master
       install:
-        - ./travis-ci/fetch-cache.sh
-        - npm install
         - ln -s /dev/stdout ./lerna-debug.log
-        - npm run bootstrap-no-build
-        - echo 'BOOTSTRAP FINISHED'
       script: ./travis-ci/travis_wait_new 50 ./travis-ci/run-redeployment-tests.sh
       after_failure: ./travis-ci/travis_wait_new 50 ./travis-ci/unlock-stack.sh
 
     - name: "Cleanup integration tests"
       stage: cleanup integration tests
       if: commit_message !~ \[skip-integration-tests\] OR branch = master
       install:
-        - npm install
+        - ln -s /dev/stdout ./lerna-debug.log
```

Comment:
What exactly does this do? Print the lerna debug output to stdout?

Reply:
From [here](https://github.com/lerna/lerna#concepts): Lerna will log to a lerna-debug.log file (same as npm-debug.log) when it encounters an error running a command.

This writes those errors out to stdout so that we can see them in Travis.